packages/react-form/src/start/utils.ts

```diff
@@ -1,3 +1,15 @@
-import { createCookie } from '@remix-run/node'
+import { getCookie, setCookie } from '@tanstack/react-start/server'
+import * as devalue from 'devalue'
 
-export const _tanstackInternalsCookie = createCookie('_tanstack_form_internals')
+export const _tanstackInternalsCookie = {
+  name: 'tanstack-internals',
```

Comment:
An extra consideration on the name declared this way in the object, is it because we want it to be accessible from the outside? The only line we need it is where we delete the cookie.

Since we're wrapping the cookie handling here it might make sense to also add the `delete` method on this object and have a non exported `const INTERNALS_COOKIE_NAME` on this file, so that then internal cookie is only handled through these methods.

What do you think?

Reply:
Thank you for your review.
I implemented this way to keep the original class structure. 
So, for improvements
- declare cookie name in `utils.ts` and we don't export it.
- create independent functions like `setTanstackInternalCookie` `getInternalCookie` and `deleteInternalCookie` 

This is my plan. If it looks good to you, I gonna start modifying the codebase

Reply:
`[set|get|delete]TanStackInternalCookie` sounds good, thank you 👍

Reply:
done on fb6a47135f23 !